dist/build.sbt

```diff
@@ -106,6 +108,23 @@ lazy val dist = project.in(file("."))
           case ((platform: PlatformBuild, subApp: SubApplication), jpkgr: BuildJDK) => (platform, subApp, jpkgr)
         }
     },
+    packagedMathematicaLink := {
+      val mathematicaLinkDir = netLogoRoot.value / "Mathematica-Link"
+      IO.createDirectory(target.value / "Mathematica Link")
+      Seq(
+        mathematicaLinkDir / "NetLogo-Mathematica Tutorial.nb",
+        mathematicaLinkDir / "NetLogo-Mathematica Tutorial.pdf",
+        mathematicaLinkDir / "NetLogo.m",
+        mathematicaLinkDir / "target" / "scala-2.10" / "mathematica-link.jar")
```

Comment:
`sbt` puts it there. What's the best practice for this situation? I would rather not specify the scala version in the `build.sbt`, since that's irrelevant to the project. I don't really care where the compiled artifact ends up, but I would prefer it be the same even when the scala version used by NetLogo changes.